Updates to support building with flutter

oxtoacart opened the pull request.

oxtoacart committed:
Updates to support building with flutter

oxtoacart commented on Dockerfile:
```diff
@@ -23,29 +25,30 @@ ENV PATH $PATH:/usr/local/gradle/bin
 # Install Android SDK. You can update this from here:
 # https://developer.android.com/studio#downloads
-ENV SDK_TOOLS commandlinetools-linux-6609375_latest.zip
+ENV SDK_TOOLS commandlinetools-linux-7302050_latest.zip
```
Need the newer commandlinetools for the updated ndk

oxtoacart commented on Dockerfile:
```diff
@@ -1,15 +1,17 @@
+# Install Java and other dev tools
 RUN apt-get install -y apt-utils
-RUN apt-get install -y openjdk-8-jdk curl wget unzip build-essential git file
+RUN apt-get install -y openjdk-8-jdk curl wget unzip build-essential git git-lfs file
```
Need git-lfs to build flashlight

atavism commented on Dockerfile:
```diff
@@ -82,3 +85,8 @@ RUN pip3 install gmsaas
+# Flutter
+RUN wget https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_2.0.6-stable.tar.xz
+RUN tar xf flutter_linux_2.0.6-stable.tar.xz
+RUN mv flutter /usr/local/
```
@oxtoacart If you want to avoid hard-coding a specific Flutter version here, you could just always obtain the latest version [from master](https://github.com/flutter/flutter) (and simplify the above to a one-liner): 

```Dockerfile
RUN git clone https://github.com/flutter/flutter.git /usr/local/flutter
```

Maybe also verify the flutter installation afterwards by running the following?

```Dockerfile
RUN flutter channel master
RUN flutter upgrade
RUN flutter doctor -v
```

atavism replied:
Oh, you also need this line:

```Dockerfile
ENV PATH $PATH:/usr/local/flutter/bin/:/usr/local/flutter/bin/cache/dart-sdk/bin
```

oxtoacart replied:
I like running flutter doctor, I'll add that. I'm intentionally pinning flutter 2.0.6, as some of our libraries ran into issues with 2.2.0.

atavism replied:
Oh gotcha, sounds good

oxtoacart committed:
Code review updates

oxtoacart commented:
Thanks for the review @atavism! I added a call to flutter doctor and also updated the environment to add flutter and dart to the path. These weren't necessary for the CircleCI build to run, but it's nice having them there.
